BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -133,6 +133,18 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
             .shared.transactionDocsDataCoordinator as? TransactionDocsDataInternalProtocol
     }
 
+    /**
+     Initializes a new instance.
```

Comment:
[Suggestion]: I think it makes sense to add `Initializes a new instance of ObjectName`